pom.xml

```diff
@@ -95,6 +95,8 @@
 
     <!-- Release: enable publication to Bintray -->
     <artifactsToPublish>${project.groupId}:sonar-java-plugin:jar</artifactsToPublish>
+    <!-- Override value from parent pom, needed to process jars compiled with Java 9 (mainly JAXB) -->
+    <version.shade.plugin>3.1.0</version.shade.plugin>
```

Comment:
I am wondering if we require this, why not upgrade the version in the parent pom ? cc @drautureau-sonarsource

Reply:
Indeed I just need to release the next version of the parent

Reply:
@drautureau-sonarsource any ETA ?

Reply:
Parent 41 is available